docs/DeveloperGuide.md

```diff
@@ -574,6 +570,32 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
     * Steps 1a1-1a2 are repeated until the increment amount entered is valid.
     * Use case resumes from step 2.
 
+**Use case: UC6 - View Command History**
+
+**MSS**
+
+1. User requests to view the command history of a certain number of past commands.
+1. EmployeeManager validates the input and fetches the required number of recent valid commands from the CommandHistory.
+1. EmployeeManager displays the fetched commands to the user.
+1. Use case ends.
```

Comment:
MSS numbering all are 1, perhaps it should be 1, 2, 3, 4?

Reply:
I just realised that a few other use cases also have the same numbering issue

Reply:
In markdown, the numbers will be auto-incremented, so putting all ones makes it easier to add new points in the middle of the list.

Reply:
oh ok!